### ISSUE TITLE
Add momentum scrolling support

### ISSUE THREAD

**Issue by rustyx** [ariutta/svg-pan-zoom#174]
Please add momentum scrolling support, it's a de-facto scrolling feature in modern web applications.

**Comment by bumbu** [comment]
Now you can simulate this using something similar to [pan animate demo](http://ariutta.github.io/svg-pan-zoom/demo/simple-animation.html).
Animated zoom looks nice, but it has a performance cost - so it could be a toggleable option.

**Comment by simonmacfadyen** [comment]
Is there an example of this using easing?

**Comment by bumbu** [comment]
@simonmacfadyen no, there is no, but you could see how easing is implemented in other libraries and use that

**Comment by simonmacfadyen** [comment]
@bumbu fair enough, I will take a look :)
